### PR TITLE
update rubygems and include pure-Ruby readline implementation

### PR DESCRIPTION
**Pull request by TheLunaticScripter**
Signed-off-by: John Snow <thelunaticscripter@outlook.com>

### CODE REVIEW

File: ruby/plan.ps1 [hunk 1]
```diff
@@ -13,7 +13,7 @@ $pkg_deps=@(
     "core/openssl"
     "core/zlib"
 )
-$pkg_build_deps=@("core/visual-cpp-build-tools-2015")
+$pkg_build_deps=@("core/visual-cpp-build-tools-2015", "core/zlib", "core/openssl")
```
Comment (robbkidd) [inline ruby/plan.ps1]: These are both listed as runtime dependencies in `pkg_deps`, so they do not need to be repeated in `pkg_build_deps`.

Comment (TheLunaticScripter) [reply ruby/plan.ps1]: Sweet yep I'll pull those out.

**Comment by robbkidd** [comment]
I did some git history rewriting. Merging!

**Comment by TheLunaticScripter** [comment]
Sweet thank you!

**Pull request merged**